minor_grammatical_edits (#219)
datacarpentry/OpenRefine-ecology-lesson: _episodes/01-working-with-openrefine.md

*Exploring data by applying multiple filters*

Facets are one of the most useful features of OpenRefine and can help get an overview of the data in a project as well as helping you bring more consistency to the data. OpenRefine supports faceted browsing as a mechanism for

* seeing the big picture of your data, and
* filtering down to just the subset of rows that you want to change in bulk.

> A 'Facet' groups all the identical values that appear in a column, and then allows you to filter the data by these values and edit values across many records at the same time.

Reviewer: ```suggestion
A 'Facet' groups the values that appear in a column according to some criterion, and then allows you to filter the groups and edit values across many records at the same time.
```
